src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx

```diff
@@ -1482,7 +1482,7 @@ namespace xclhwemhal2 {
 
     if (xclemulation::config::getInstance()->isFastNocDDRAccessEnabled())
     {
-      std::cout << "Checking Write Fastmem " << dest << std::endl;
+      // std::cout << "Checking Write Fastmem " << dest << std::endl;
       if (mNocFastAccess.isAddressMapped(dest, size))
       {
         std::cout << "Writing Fastmem " << dest << std::endl;
```

Comment:
Is there any reason why we did not remove this std::cout? Please remove even this

Reply:
Removed.